app/experimenter/static/rapid/components/experiments/ExperimentEditHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+
+import { useExperimentState } from "experimenter-rapid/contexts/experiment/hooks";
+
+const ExperimentFormPage: React.FC = () => {
+  const { location } = useHistory();
+  let pageHeading = "Create a New A/A Experiment";
+  if (location.pathname.includes("edit")) {
```

Comment:
Can't we look at the experiment data to see if it has a name instead of looking at the url?

Reply:
so... i took a stab at only using experiment data. Since the name data value changes when a user changes the name in the text field, I couldn't think of a way how would that indicate whether the experiment was new, or an edit. I decided using the presence of a slug in the experiment data instead?